core/src/main/resources/hudson/tasks/ArtifactArchiver/help-artifacts.html

```diff
@@ -5,4 +5,11 @@
    -- except that <tt>","</tt> (<em>comma</em>) is the only supported separator.
    The base directory is <a href='ws/'>the workspace</a>.
    You can only archive files that are located in your workspace.
+<br/>
+   An example on how to archive multiple artifacts from a specific folder:
+   <pre>archiveArtifacts artifacts: 'target/*.jar'</pre>
+<br/>
+   An example on how to archive multiple artifacts with different patterns:
+   <pre>archiveArtifacts: artifacts: 'target/*.jar, target/*.war'</pre>
```

Comment:
Use the same syntax in both examples (omit the extra ':')
```suggestion
   <pre>archiveArtifacts artifacts: 'target/*.jar, target/*.war'</pre>
```